engine/src/main/java/org/terasology/network/internal/NetworkSystemImpl.java

```diff
@@ -150,6 +150,7 @@
     private Map<EntityRef, EntityRef> ownerLookup = Maps.newHashMap();
     private SetMultimap<EntityRef, EntityRef> ownedLookup = HashMultimap.create();
     private StorageManager storageManager;
+    private String errorMessage;
```

Comment:
I'm not 100% sure but it seems like it's not strictly necessary to have this as a global field rather than a local variable passed around when needed. I think it would need to be passed just to `NetData.ServerInfoMessage getServerInfoMessage()`. So it may become a `private NetData.ServerInfoMessage getServerInfoMessage(String errorMessage)`, then to avoid breaking the interface of the class we could add an overloaded version with the same signature as before, like this:
```
NetData.ServerInfoMessage getServerInfoMessage() {
    return getServerInfoMessage(null);
}
```

Reply:
Good point, this isn't too hard to fix so I will do so.